src/background/background-init.ts

```diff
@@ -37,21 +36,22 @@ import { getTelemetryClient } from './telemetry/telemetry-client-provider';
 import { TelemetryEventHandler } from './telemetry/telemetry-event-handler';
 import { TelemetryLogger } from './telemetry/telemetry-logger';
 import { TelemetryStateListener } from './telemetry/telemetry-state-listener';
-import { UserStoredDataCleaner } from './user-stored-data-cleaner';
+import { cleanKeysFromStorage } from './user-stored-data-cleaner';
 
 declare var window: Window & InsightsFeatureFlags;
 
 const browserAdapter = new ChromeAdapter();
 const urlValidator = new UrlValidator(browserAdapter);
-const backgroundInitCleaner = new UserStoredDataCleaner(browserAdapter);
 const indexedDBInstance: IndexedDBAPI = new IndexedDBUtil(getIndexedDBStore());
 const indexedDBDataKeysToFetch = [IndexedDBDataKeys.assessmentStore, IndexedDBDataKeys.userConfiguration];
 
-backgroundInitCleaner.cleanUserData(deprecatedStorageDataKeys);
+cleanKeysFromStorage(browserAdapter, deprecatedStorageDataKeys).catch(reason => console.error('error while cleaning user data: ', reason));
 
-// tslint:disable-next-line:no-floating-promises - top-level entry points are intentionally floating promises
-getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then((persistedData: PersistedData) => {
-    browserAdapter.getUserData(storageDataKeys, (userData: LocalStorageData) => {
+const persistedDataPromise = getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch);
+const userDataPromise = browserAdapter.getUserData(storageDataKeys);
```

Comment:
should we get user data after the deprecated keys are removed since removing deprecated keys is a async call? (I know you didnt change the behavior. it could be handled in a later pr)

Reply:
That's a good point!
I think is a minor improvement though is I did not find any other reference to the deprecated keys expect for the removal call. I'll leave that for a future PR.